pkg/controller/installation/products/monitoring/reconciler.go

```diff
@@ -2,7 +2,11 @@ package monitoring
 
 import (
 	"context"
+	"encoding/json"
 	"fmt"
+	"github.com/operator-framework/operator-marketplace/pkg/client"
+	v12 "k8s.io/api/core/v1"
```

Comment:
`corev1` would have been better, but we can address this later (I created [a JIRA](https://issues.jboss.org/browse/INTLY-4187) about unifying naming in this operator).